macantine/urls.py

```diff
@@ -8,11 +8,13 @@
 from wagtail.admin import urls as wagtailadmin_urls
 from wagtail import urls as wagtail_urls
 from wagtail.documents import urls as wagtaildocs_urls
+from cms.api import api_router
 
 urlpatterns = [
     path("admin/", admin.site.urls),  # if the path of 'admin/' changes, update historical_record_add_auth_method
     path("ckeditor/", include("ckeditor_uploader.urls")),
     path("o/", include("oauth2_provider.urls", namespace="oauth2_provider")),
+    path("api/v2/", api_router.urls),
```

Comment:
```suggestion
    path("apicms/v1/", api_router.urls),
```
`api/v2/` est reservé pour si on MAJ notre api qui existe déjà.